docs/spectral-rules/index.md

```diff
@@ -61,6 +61,13 @@ The following is a sample Github actions job which can be used as an example of
 
 ```yaml
 ...
+---
+name: API Standards Checks
+
+on:
+  pull_request:
+    branches: [main, master]
```

Comment:
can we add a comment here to say this is just an example you wouldn't normally have a `main` and a `master branch